### PR TITLE
Method name optimization

### PR DESCRIPTION
**Pull request by pengzhengfa**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by pengzhengfa** [comment]
@KomachiSion Hello, I have optimized the name of the hump in the method name of the main logic, could you please have a look

**Comment by pengzhengfa** [comment]
@KomachiSion  Hello, I may have some spaces when Reformat the code

**Comment by KomachiSion** [comment]
@pengzhengfa Please use `nacos-code-style-for-idea.xml` to reformat code.

This file is under dir of source code `style`

**Comment by pengzhengfa** [comment]
ok

**Comment by KomachiSion** [comment]
> ok

Please see `style/codeStyle.md`.

If you use eclipse, Please only change these code you want to changed.

And we are welcome to contribute eclipse code style file according to `nacos-code-style-for-idea.xml`

**Comment by pengzhengfa** [comment]
@KomachiSion Hello, I have solved it. Please check

File: naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java [hunk 1]
```diff
@@ -312,7 +312,7 @@ public void updateIps(List<Instance> ips, boolean ephemeral) {
         
         Map<String, Integer> intersectMap = new ConcurrentHashMap<>(newInstance.size() + oldInstance.size());
         Map<String, Instance> instanceMap = new ConcurrentHashMap<>(newInstance.size());
-        Map<String, Instance> instanceMap1 = new ConcurrentHashMap<>(newInstance.size());
+        Map<String, Instance> instancesMap = new ConcurrentHashMap<>(newInstance.size());
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java:315]: I read method again and I think we should change `instanceMap` to `updatedInstancesMap`
change `instanceMap1 ` to `newInstancesMap`

File: naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java [hunk 2]
```diff
@@ -329,8 +329,8 @@ public void updateIps(List<Instance> ips, boolean ephemeral) {
                     intersectMap.put(instance.toString(), 1);
                 }
             }
-            
-            instanceMap1.put(instance.toString(), instance);
+
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java:332]: Do not change the Indentation.

File: naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java (hunk 1 above)
Comment (pengzhengfa) [reply naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java:315]: Ok, I change

File: naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java (hunk 2 above)
Comment (pengzhengfa) [reply naming/src/main/java/com/alibaba/nacos/naming/core/Cluster.java:332]: Ok, I change

**Pull request merged**